tests/acceptance/features/apiUserLDAPConnection/backupServerConnection.feature

```diff
@@ -0,0 +1,56 @@
+@api
+Feature: connect to a backup LDAP serer
+
+As an administrator
+I want to be able to set up a backup LDAP server
+So that user authentifications still work when the main LDAP server is not reachable
```

Comment:
s/authentifications still work/authentication still works/

"authentification" is not an English word

Reply:
done